iot/api-client/beta-features/commands/receive/resources/rsa_private.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Do we need to check in a private key? Seems like encouraging a bad practice.

Reply:
It's used in testing / device should be de-authorized after tests complete